public/app/percona/pmm-dump/components/ExportDataset/ExportDataset.tsx

```diff
@@ -0,0 +1,225 @@
+import { useHistory } from 'react-router-dom';
+
+import { SelectableValue, DateTime, dateTime, TimeRange, AppEvents } from '@grafana/data';
+import { LinkButton, PageToolbar, DateTimePicker, useStyles2 } from '@grafana/ui';
+import appEvents from 'app/core/app_events';
+import { GrafanaRouteComponentProps } from 'app/core/navigation/types';
+import { LoaderButton } from 'app/percona/shared/components/Elements/LoaderButton';
+import { Overlay } from 'app/percona/shared/components/Elements/Overlay';
+import { MultiSelectField } from 'app/percona/shared/components/Form/MultiSelectField';
+import { useCancelToken } from 'app/percona/shared/components/hooks/cancelToken.hook';
+import { fetchActiveServiceTypesAction, fetchServicesAction } from 'app/percona/shared/core/reducers/services';
+import { getServices } from 'app/percona/shared/core/selectors';
+import { isApiCancelError } from 'app/percona/shared/helpers/api';
+import { useAppDispatch } from 'app/store/store';
+import { useSelector } from 'app/types';
+
+import { SwitchRow } from '../../../settings/components/Advanced/SwitchRow';
+import { PMMDumpService } from '../../PMMDump.service';
+
+import { GET_SERVICES_CANCEL_TOKEN, DUMP_URL } from './ExportDataset.constants';
+import { Messages } from './ExportDataset.messages';
+import { getStyles } from './ExportDataset.styles';
+import { ExportDatasetProps } from './ExportDataset.types';
+
+const { Form } = withTypes<ExportDatasetProps>();
+
+const ExportDataset: FC<GrafanaRouteComponentProps<{ type: string; id: string }>> = ({ match }) => {
+  const styles = useStyles2(getStyles);
+  const dispatch = useAppDispatch();
+  const [selectedTimerange, setSelectedTimerange] = useState<TimeRange>();
+  const { isLoading, services: fetchedServices } = useSelector(getServices);
+
+  const flattenServices = useMemo(
+    () =>
+      fetchedServices.map((data: { params: { serviceName: string } }): SelectableValue<string> => {
+        return {
+          label: data.params.serviceName,
+          value: data.params.serviceName,
+        };
+      }),
+    [fetchedServices]
+  );
+
+  const [generateToken] = useCancelToken();
+  const [endDate, setEndDate] = useState<DateTime>(dateTime(new Date().setSeconds(0, 0)));
+  let defaultEndDate = new Date();
+  defaultEndDate.setSeconds(0, 0);
+  const [date, setDate] = useState<DateTime>(dateTime(new Date(defaultEndDate.getTime() - 6 * 1000 * 60 * 60)));
+
+  const loadData = useCallback(async () => {
+    try {
+      await dispatch(fetchServicesAction({ token: generateToken(GET_SERVICES_CANCEL_TOKEN) }));
+      await dispatch(fetchActiveServiceTypesAction());
+    } catch (e) {
+      if (isApiCancelError(e)) {
+        return;
+      }
+      console.log(e);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  useEffect(() => {
+    loadData();
+    setSelectedTimerange({
+      from: dateTime(new Date('2023-10-03T04:35:46.795Z')),
```

Comment:
What's the reason for hardcoding it here?

Reply:
Updated.